Dockerfile

```diff
@@ -63,10 +67,10 @@ COPY insurance-requirements.txt buildreqs/insurance-requirements.txt
 RUN apt-get update && apt-get install -y software-properties-common && apt-get update
 
 RUN add-apt-repository ppa:deadsnakes/ppa
-RUN apt-get update
 
 # Install python 3.5 from deadsnakes
-RUN apt-get install -y libpq-dev build-essential python3.5 python3.5-dev python3-pip python3.5-venv
+RUN apt-get update && apt-get install -y libpq-dev build-essential python3.5 python3.5-dev python3-pip python3.5-venv
+RUN apt-get update && add-apt-repository ppa:deadsnakes/ppa && apt-get install -y libpython3.5-tk
```

Comment:
I had to add this line to install libpython3.5-tk in order to fix the _tkinter import error.